### ISSUE TITLE
ModuleNotFoundError: No module named 'pip'

### ISSUE THREAD

**Issue by Aransh** [maxileith/homebridge-appletv-enhanced#821]
### Preconditions

- [X] Requirements fulfilled
- [X] The bug is not a known issue
- [X] The bug has not been solved in the past or the solution that was provided in the past does not work on my system
- [X] There is currently no other opened issue that describes this bug
- [X] I cannot find any solution to the problem in GitHub discussions
- [X] I am certain that the problem that I am having is a bug

### Analysis

Plugin fails on start, giving error “ModuleNotFoundError: No module named 'pip'”.
Been using the plugin for months, so this of a new issue

### Expected Behavior

Plugin successfully starts

### Steps To Reproduce

1. Install latest homebridge docker image
2. Install plugin

### Logs

```console
[12/20/2024, 2:34:24 PM] [Apple TV Enhanced] Initializing AppleTVEnhanced platform...
[12/20/2024, 2:34:24 PM] [Apple TV Enhanced] [I] Platform: Finished initializing platform: Apple TV Enhanced
[12/20/2024, 2:34:27 PM] [Apple TV Enhanced] [I] Update check: The update checker is configured to check for updates between 3:00 and 3:59, excluding betas. Auto updating is turned off.
[12/20/2024, 2:34:31 PM] [Apple TV Enhanced] [S] Update check: You are using the latest version of AppleTV Enhanced (excluding betas): 1.10.2
[12/20/2024, 2:34:31 PM] [Apple TV Enhanced] [I] Python check: Starting python check.
[12/20/2024, 2:34:31 PM] [Apple TV Enhanced] [I] Python check: Using "python3" as the python executable.
[12/20/2024, 2:34:31 PM] [Apple TV Enhanced] [I] Python check: plugin dir exists.
[12/20/2024, 2:34:31 PM] [Apple TV Enhanced] [I] Python check: OpenSSL 3.0.13 is installed and compatible.
[12/20/2024, 2:34:31 PM] [Apple TV Enhanced] [I] Python check: Python 3.12.3 is installed and supported by the plugin.
[12/20/2024, 2:34:31 PM] [Apple TV Enhanced] [I] Python check: Virtual environment already exists.
[12/20/2024, 2:34:34 PM] [Apple TV Enhanced] [I] Python check: Virtual environment is using the configured python environment. Continuing ...
[12/20/2024, 2:34:34 PM] [Apple TV Enhanced] [E] Python check: Traceback (most recent call last):
  File "/var/lib/homebridge/appletv-enhanced/.venv/bin/pip3", line 5, in <module>
    from pip._internal.cli.main import main
ModuleNotFoundError: No module named 'pip'
[12/20/2024, 2:34:34 PM] [Apple TV Enhanced] [I] Python check: Venv pip version: 
[12/20/2024, 2:34:34 PM] [Apple TV Enhanced] [I] Python check: Checking if there is an update for venv pip ...
[12/20/2024, 2:34:35 PM] [Apple TV Enhanced] [W] Python check: Venv pip is outdated. Updating now ...
[12/20/2024, 2:34:35 PM] [Apple TV Enhanced] [E] Python check: Traceback (most recent call last):
  File "/var/lib/homebridge/appletv-enhanced/.venv/bin/pip3", line 5, in <module>
    from pip._internal.cli.main import main
ModuleNotFoundError: No module named 'pip'
[12/20/2024, 2:34:35 PM] [Apple TV Enhanced] [W] Python check: Failed to update venv pip. Continuing anyhow ...
[12/20/2024, 2:34:35 PM] [Apple TV Enhanced] [E] Python check: Traceback (most recent call last):
  File "/var/lib/homebridge/appletv-enhanced/.venv/bin/pip3", line 5, in <module>
    from pip._internal.cli.main import main
ModuleNotFoundError: No module named 'pip'
[12/20/2024, 2:34:35 PM] [Apple TV Enhanced] [W] Python check: Python requirements are not satisfied. Installing them now ...
[12/20/2024, 2:34:35 PM] [Apple TV Enhanced] [E] Python check: Traceback (most recent call last):
  File "/var/lib/homebridge/appletv-enhanced/.venv/bin/pip3", line 5, in <module>
    from pip._internal.cli.main import main
ModuleNotFoundError: No module named 'pip'
[12/20/2024, 2:34:35 PM] [Apple TV Enhanced] [E] Python check: There was an error installing the python dependencies. Cannot proceed!
```

### Configuration

```JSON
{
  "absoluteVolumeControl" : false,
  "avadaKedavraAppAmount" : 15,
  "customInputURIs" : [

  ],
  "deviceSpecificOverrides" : [
    {
      "avadaKedavraAppAmount" : 15,
      "deviceStateDelay" : 0,
      "disableVolumeControlRemote" : false,
      "overrideAvadaKedavraAppAmount" : false,
      "overrideCustomInputURIs" : false,
      "overrideDeviceStateDelay" : false,
      "overrideDeviceStates" : false,
      "overrideDisableVolumeControlRemote" : false,
      "overrideMediaTypes" : false,
      "overrideRemoteKeysAsSwitch" : false,
      "overrideSetTopBox" : false,
      "setTopBox" : false
    }
  ],
  "deviceStateDelay" : 0,
  "deviceStates" : [

  ],
  "disableVolumeControlRemote" : false,
  "discover" : {
    "blacklist" : [

    ],
    "multicast" : true,
    "unicast" : [

    ]
  },
  "forceVenvRecreate" : false,
  "logLevel" : 3,
  "mediaTypes" : [

  ],
  "name" : "Apple TV Enhanced",
  "platform" : "AppleTVEnhanced",
  "remoteKeysAsSwitch" : [

  ],
  "setTopBox" : false,
  "updateCheckLevel" : "stable",
  "updateCheckTime" : 3
}
```

### Operating System

Linux

### Operating System: Bits

64-bit

### Operating System: Distribution

Ubuntu (Running in docker on Raspberry Pi OS)

### Operating System: Distribution Version

24.04.1 LTS (Noble Numbat)

### Docker

yes

### Docker Image

homebridge/homebridge

### Docker Image Tag

2024-12-19

### Homebridge Version

1.8.5

### Homebridge Config UI Version

4.65.2

### Homebridge Storage Path

/var/lib/homebridge

### Homebridge Apple TV Enhanced Version

1.10.2

### Node Version

v22.12.0

### NPM Version

10.9.0

### Python Version

3.12.3

### PIP Version

24.0.0

### HDMI Hops

None

### Audio Output

no

### Same Subnet

yes

### Additional Context

Issue started after updating homebridge docker image

**Comment by Aransh** [comment]
Forced venv recreate and working again

**Comment by maxileith** [comment]
For everyone that has the same problem and stumbles up on this issue ... the issue has been fixed in v1.10.3.
